### PR TITLE
Fix sameSite (Traefik v1)

### LINKED ISSUE #6261: Allow configuration of SameSite settings for sticky session cookies
Feature
With Chrome 80 going stable on Feb 4, 3rd party cookies without the `SameSite=None` setting will be blocked. https://blog.chromium.org/2019/10/developers-get-ready-for-new.html. In our testing, this breaks sticky sessions for services that are iframed into another page.

Relates to #2653, #4738

### PR DESCRIPTION
**Pull request by ldez**
### What does this PR do?

Allow to configure cookie options:
- sameSite
- httpOnly
- secure

### Motivation

Fixes #6261

### More

- [x] Added/updated tests
- [x] Added/updated documentation

### Additional Notes

~~Waiting for https://github.com/vulcand/oxy/pull/195~~

### CODE REVIEW

File: docs/basics.md [hunk 1]
```diff
@@ -452,6 +452,28 @@ If not, a new backend will be assigned.
...
+    # Optional
+    # Default: false
+    #
+    #  httpOnly = true
+
+    # Customize same site option.
+    # Can be: none, lax, strict
+    #
+    # Optional
+    # Default:
```
Comment (jbdoumenjou) [inline docs/basics.md]: "" as Default ?

File: docs/basics.md [hunk 2]
```diff
@@ -452,6 +452,28 @@ If not, a new backend will be assigned.
...
+    #
+    #  httpOnly = true
+
+    # Customize same site option.
+    # Can be: none, lax, strict
+    #
+    # Optional
+    # Default: 
+    #
+    #  httpOnly = true
```
Comment (jbdoumenjou) [inline docs/basics.md]:
```suggestion
    #  sameSite = "none"
```

**Comment by mayrbenjamin92** [comment]
Hi guys!

I can see that the fix was already merged - but I don't understand in which 1.7.X release this will be integrated. We are (still) using the 1.7 product-line in a large enterprise-product and need to understand when this will land. I would extremely appreciate any feedback.

Thanks for this awesome product and your hard work!

**Comment by ldez** [comment]
https://github.com/containous/traefik/blob/v1.7/CHANGELOG.md

v1.7.23

**Comment by mayrbenjamin92** [comment]
One question related to the same-site fix - I have checked the documentation:
https://docs.traefik.io/v1.7/basics/#sticky-sessions

We are using docker-swarm together with Traefik - but I seem to have issues with adding the necessary labels to the swarm service in my compose file. The documentation (check the referenced link) shows how to configure same-site if the file-provider is used - I am unable to properly "translate" that to the labels for my swarm service.

Does anybody of you guys have an example?

**Pull request merged**